src/acf.py

```diff
@@ -122,12 +129,27 @@ def disconnect(self):
         except:
             pass
 
+    def timer_callback(self):
+        self.send_command(self.force)
+        self.initial_force = self.force
+        telem = self.handle_telem()
+        if self.joint_state_pub is not None:
+            msg = JointState()
+            msg.header.stamp = self.get_clock().now().to_msg()
+            msg.header.frame_id = f"{self.i}"
```

Comment:
The `frame_id` field is intended to capture the TF frame to be associated with the data published. For instance a `link` in a URDF.

This seems to use it as a sequence nr. I'd suggest not to do that.

But having written that: `frame_id` is not almost never used when publishing `JointState` messages, as the name(s) of the joint(s) already encode the 'location' to associate with the sensor data.

Reply:
This was a misunderstanding from my side, I was thinking to use `seq`, which is no longer available in ROS2 apparently.

Reply:
Fixed in [f3ce32a](https://github.com/sam-xl/ferrobotics_acf/pull/2/commits/f3ce32aa2fc05ab5887ebff890226e9d339942ac)